ckan/config/middleware/flask_app.py

```diff
@@ -160,10 +160,11 @@ def make_flask_stack(conf: Union[Config, CKANConfig]) -> CKANApp:
 
     # Update Flask config with the CKAN values. We use the common config
     # object as values might have been modified on `load_environment`
-    if config:
-        app.config.update(config)
-    else:
-        app.config.update(conf)
+    app.config.update({
+        k: config[k]
+        for key in config_declaration.iter_options()
+        if config_declaration[key].has_flag(Flag.flask) and (k := str(key))
```

Comment:
Trying to understand `and (k := str(key)` here because it looks funny.

Can a config key ever not be a string? Can it ever be `""`?